modules/ossm-threescale-integrate.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/threescale-adapter.adoc
+
+[id="ossm-threescale-integrate_{context}"]
+= Integrate the 3Scale adapter with {ProductName}
+You can use these examples to configure requests to your services using the 3scale Istio Adapter.
+
+
+.Prerequisites:
+
+* {ProductName} 0.7.0+
+* A working 3scale account (link:https://www.3scale.net/signup/[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.5/html/installing_3scale/onpremises-installation[3scale 2.5 On-Premises])
+* xref:../service_mesh_install/preparing-ossm-installation.html#ossm-install-prerequisites_preparing-ossm-installation[{ProductName} prerequisites]
+* Ensure Mixer policy enforcement is enabled. xref:../service_mesh_install/installing-ossm.html#ossm-mixer-policy_installing-ossm[Update Mixer policy enforcement] provides instructions to check the current Mixer policy enforcement status and enable policy enforcement.
+
+[NOTE]
+====
+To configure the 3scale Istio Adapter, refer to xref:../service_mesh_install/installing-ossm.html#ossm-cr-example_installing-ossm[{ProductName} custom resources] for instructions on adding adapter parameters to the custom resource file.
+====
+
+
+[NOTE]
+====
+Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the service ID of the API you want to manage.
+====
+
+. Modify the handler configuration with your 3scale configuration.
++
+[source,yaml]
+----
+  apiVersion: "config.istio.io/v1alpha2"
+  kind: handler
+  metadata:
+   name: threescale
+  spec:
+   adapter: threescale
+   params:
+     service_id: "<SERVICE_ID>"
+     system_url: "https://<organization>-admin.3scale.net/"
+     access_token: "<ACCESS_TOKEN>"
+   connection:
+     address: "threescale-istio-adapter:3333"
+----
+
++
+. Create the required resources:
++
+----
+oc create -f istio/
```

Comment:
All CLI commands need to start with $ or #. # means that you need root access.

Reply:
Fixed this for the 3scale team.